target/linux/ath79/dts/ar7161_mikrotik_routerboard-493g.dts

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-1.0-or-later OR MIT
+
+// Based on ar71xx arch/mips/ath79/mach-rb4xx.c
```

Comment:
Remove that, maybe add a comment to the commit message, but I'd remove it entirely.

Reply:
Deleted